pkg/client/client.go

```diff
@@ -0,0 +1,41 @@
+package client
+
+import (
+	"fmt"
+	"os"
+	"os/user"
+	"path/filepath"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// GetConfig creates a *rest.Config for talking to a Kubernetes apiserver.
+// Otherwise will assume running in cluster and use the cluster provided kubeconfig.
+//
+// Config precedence
+//
+// * KUBECONFIG environment variable pointing at a file
+//
+// * In-cluster config if running in cluster
+//
+// * $HOME/.kube/config if exists
+func GetConfig() (*rest.Config, error) {
```

Comment:
I'm guessing this is based on code from somewhere?  Is `client.go` the best package name? Just thinking about it as it only contains `GetConfig()`.

Reply:
yes, its from https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/client/client.go#L35-L53

Was thinking about creating a helper for the e2e tests but this might come handy in the codebase as well. Wasnt really sure. But I guess I can put it into the testframework. Thoughts ? :)

Reply:
Wondering if we should just copy the whole file, if the 3-4 other functions help clarify intent.  Hmm.